pages/common/pipenv.md

```diff
@@ -0,0 +1,28 @@
+# pipenv
+
+> Python Development Workflow for Humans.
```

Comment:
Perhaps this would read better in sentence case rather than title case?

Reply:
I agree